### ISSUE TITLE
LC0012 reporting false positives for all event subscribers

### ISSUE THREAD

**Issue by rvanbekkum** [StefanMaron/BusinessCentral.LinterCop#72]
Hi @StefanMaron,

Running into issues with rule LC0012 which reports a problem for any event subscriber.

**Example:**

```al
    [EventSubscriber(ObjectType::Codeunit, Codeunit::"Guided Experience", 'OnRegisterAssistedSetup', '', false, false)]
    local procedure CodeunitGuidedExperience_OnRegisterAssistedSetup()
    begin
        /// ... some statement that invokes a procedure
    end;
```

```
Wrong Parameter detected. Select the correct object with "ObjectType::Codeunit;Codeunit::"Guided Experience"::" instead.
```
